requirements.txt

```diff
@@ -24,6 +24,7 @@ typing-extensions==3.7.4
 urllib3==1.25.6
 Werkzeug==0.16.0
 serial==0.0.97
```

Comment:
not sure, should be only `pyserial`

Reply:
Ups, sorry, missed your comment. I'll fix it with the next change!